utility-types/Exercise1/solution/Solution.ts

```diff
@@ -1,39 +1,47 @@
 /*
-1)
-
-Using partials is helpful for many reasons, including:
-
-Partial updates sent to PATCH endpoints
-
-Relaxing constraints when writing test
-
+Use the appropriate utility type to address the comments below.
 */
 
-//2)
-
-type Person =
-  | {
-      role: "developer"
-      email: string
-      id: number
-      firstName: string
-      lastName: string
-      team: "React" | "Angular" | "backend"
-    }
-  | {
-      role: "user"
-      email: string
-      id: number
-      firstName: string
-      lastName: string
-      isVerified: boolean
-    }
-type ProfileData = Partial<Person>
-
-const myProfile: ProfileData = {
-    email: 'email@developer.com',
-    firstName: 'Dev',
-    lastName: 'Eloper',
-}
-
-export {}
\ No newline at end of file
+type Person = {
+  role: "developer";
+  email?: string;
+  id: number;
+  firstName: string;
+  lastName: string;
+  team: "React" | "Angular" | "backend";
+};
+
+// 1. Which utility type can we use to get rid of the errow below?
+type UpdateablePerson = Partial<Person>;
+
+// This should not give an error!
+const personToUpdate: UpdateablePerson = {
```

Comment:
This still shows a warning because it’s not used—I wonder if we should add a `console.info` to the end of this exercise, just so no “problems” show up in CodeSandbox.